core/base/Dockerfile

```diff
@@ -8,11 +8,13 @@ ENV TZ=Etc/UTC LANG=C.UTF-8
 
 ARG MAILU_UID=1000
 ARG MAILU_GID=1000
+ARG TARGETPLATFORM
 
 RUN set -euxo pipefail \
   ; addgroup -Sg ${MAILU_GID} mailu \
   ; adduser -Sg ${MAILU_UID} -G mailu -h /app -g "mailu app" -s /bin/bash mailu \
   ; apk add --no-cache bash ca-certificates curl python3 tzdata \
+  ; if [[ "$TARGETPLATFORM" == 'linux/arm/v7' ]]; then exit;fi \
   ; machine="$(uname -m)" \
   ; if [[ "${machine}" == x86_64 || "${machine}" == armv8* || "${machine}" == aarch64 ]] \
   ; then \
```

Comment:
I think this early exit will confuse people adding more commands at the end.
Why not just do it like this:
```
  ; [[ "$TARGETPLATFORM" != 'linux/arm/v7' ]] \
    && machine="$(uname -m)" \
    && [[ "${machine}" == x86_64 || "${machine}" == armv8* || "${machine}" == aarch64 ]] \
    && apk add --no-cache --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing hardened-malloc
```

Reply:
Because the shell isn't bash, it's sh

Reply:
But even with the SH syntax, the ``set -euxo pipefail`` will produce an error as test returns false

Reply:
I don't see bash specifics in my proposal. But yes, it needs to be negated:
```
  ; machine="$(uname -m)"
    ! [[ "${TARGETPLATFORM}" != linux/arm/v7 && ( "${machine}" == x86_64 || "${machine}" == armv8* || "${machine}" == aarch64 ) ]] \
    || apk add --no-cache --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing hardened-malloc
```

If you don't like this please wrap the TARGETPLATFORM if/then/fi around the other lines, and don't exit early.